servercfg/serverconf.go

```diff
@@ -442,6 +452,7 @@ func GetPublicIP() (string, error) {
 				continue
 			}
 			endpoint = string(bodyBytes)
+			fmt.Println("Public IP address is", endpoint)
```

Comment:
lighter.Log()

Reply:
Thanks Matt, would like to. We're working on how to get that done.